src/scripts/Controller.js

```diff
@@ -651,6 +653,13 @@ export default class SitemapController {
 		this.setActiveNavigationButton('sitemaps');
 
 		const sitemaps = await this.store.getAllSitemaps();
+		sitemaps.forEach(sitemap => {
+			if (!sitemap.extensionVersion || sitemap.extensionVersion < 1) {
+				this.transformOldToNew(sitemap);
+				this.store.saveSitemap(sitemap);
+			}
+		});
```

Comment:
Ok, now every sitemap is converted to uuids. Next, we should think of further sitemap spec evolutions.

Suppose in the next update I want to rename uuid field to uid or id (since it is not universally unique anymore) and id field to name. How would we implement that?

Take a look at database migrations in crawlers-api. For each schema change there is a script applying it to db. We should design something similar here.

Though I'm ok with postponing it and moving to separate issue. Discuss it with @Yatskov Alex, please.